general/solomiia_tymoshchuk/Task2.java

```diff
@@ -0,0 +1,42 @@
+package general.solomiia_tymoshchuk;
+
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts a distance in meters and time in seconds and outputs a speed in meters/second and kilometers/hour.
+ * The result should be a floating point number.
+ **/
+public class Task2 {
+    double distance;
+    double time;
+
+    public Double convertSpeedValuetoMs(double distance, double time) {
```

Comment:
Since we're using the camel case per the convention, it's better to name method ...Value`To`Ms